Source/WebCore/platform/network/curl/CurlContext.cpp

```diff
@@ -381,7 +381,10 @@ CURLcode CurlHandle::willSetupSslCtxCallback(CURL*, void* sslCtx, void* userData
 
 int CurlHandle::sslErrors() const
 {
-    return m_sslVerifier ? m_sslVerifier->sslErrors() : 0;
+    if (auto verifyResult = getSSLVerifyResult(); *verifyResult != X509_V_OK)
```

Comment:
`getSSLVerifyResult` returns `std::optional<long>`. You have to check `verifyResult` before evaluating `*verifyResult`.
> if (auto verifyResult = getSSLVerifyResult(); verifyResult && *verifyResult != X509_V_OK)